src/components/TitleBar.js

```diff
@@ -22,6 +22,7 @@ const TitleBar = ({ id, onPrevious, title, onDismiss, titleExtras, style = {}, t
       onDismiss && h(Link, {
         'aria-label': 'Close',
         style: { marginLeft: '2rem' },
+        tabIndex: hideCloseButton ? -1 : 0,
```

Comment:
It really would be better to not render this component at all. If you are going to render it, you need to mark it `aria-disabled` when `hideCloseButton` is true so that the screen reader does not traverse it in the dom (I verified that I could still navigate to the close button and trigger it with the screen reader).